src/main/java/com/thebois/views/IProjector.java

```diff
@@ -0,0 +1,20 @@
+package com.thebois.views;
+
+import com.badlogic.gdx.math.Vector2;
+
+/**
+ * Interface for a Projector.
+ */
```

Comment:
You can do better than this. 😛

Reply:
Should probably have waited with writing the javadoc, and not do it at the end of the day. Changing it to something better.